### ISSUE TITLE
Stopped fecthing emails, but 

### ISSUE THREAD

**Issue by amityweb** [freescout-helpdesk/freescout#243]
Freescount stopped fetching emails today. We have lot in our email account, but nothing coming into Freescout. 

Cron is all running OK. 
> freescout:fetch-emails - Last successful run: Feb 12, 2019 13:01
> queue:work - Running

We did a manual fetch and they pulled in. 

How can we start to debug this to find out why its not pulling them in automatically? 

Thanks

**Comment by KrzysztofCW** [comment]
I have similar problem. What version php and mysql do you have?

**Comment by freescout-helpdesk** [comment]
Any messages in fetch logs?

**Comment by KrzysztofCW** [comment]
At my place:

In first mailbox (now emails not fetching, but it worked some time):
error |   | 2019-02-13 07:58:07 | Missing parameter; File: /var/www/html/app/Console/Commands/FetchEmails.php (151)

In the second (now emails not fetching, but it worked some time):
error |   | 2019-02-13 07:58:04 | iconv(): Wrong charset, conversion from `X-UNKNOWN' to `UTF-8//IGNORE' is not allowed; File: /var/www/html/vendor/webklex/laravel-imap/src/IMAP/Query/Query.php (189)

Third mailbox work fine, configuration very similar (Third mailbox work fine, configuration similar to other mailboxes: the same protocol, server, port, encryption).

**Comment by amityweb** [comment]
Found my issue. 

I had this error when running manually:
```
[root@monitor ~]# php /home/freescout/public_html/artisan schedule:run
PHP Parse error:  syntax error, unexpected 'class' (T_CLASS), expecting id
```

I have multiple PHPs on the server, calling php -v shows PHP 5.4 which is the system PHP. Now all worked fine up to the the other day, but the error above is due to atrisan not supporting PHP 5.4 So I can only imagine it was recently auto updated as it worked before, then just stopped. 

So in my case I just have to call the artisan script using PHP 7 as follows:
`/opt/rh/rh-php70/root/usr/bin/php /home/freescout/public_html/artisan schedule:run`

**Comment by KrzysztofCW** [comment]
What version of linux do you use?

**Comment by amityweb** [comment]
Centos 6 using Webmin and Virtualmin, which allows additional PHP versions installed. So the included PHP is PHP 5.3 I think but I have added PHP 7. Freescout is set to use PHP 7. But when I run "php" from the CLI, it will default to using PHP 5.3 the server base PHP. But it worked OK up to the other day when it stopped. But making sure to call it using PHP 7 is OK now.

**Comment by freescout-helpdesk** [comment]
We are glad you found a solution.
